.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
     - 5.6
     - 5.5
     - 5.4
+    - 5.3.3
```

Comment:
well, then it should be tested on both. Hopefully people still using 5.3 are at least on a recent 5.3 version